src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -29,41 +31,79 @@ public class TypicalPersons {
+            .withPhone("95352563")
+            .withEmail("heinz@example.com")
+            .withTelegramHandle("@carlkurz")
+            .withModuleName("GEA1000")
+            .withContactType("Work").build();
+    public static final Person DANIEL = new PersonBuilder().withName("Daniel Meier")
+            .withPhone("87652533")
+            .withEmail("cornelia@example.com")
+            .withTelegramHandle("@danielmeier")
+            .withTags("friends")
+            .withModuleName("HSI1000")
             .withContactType("Personal").build();
-    public static final Person ELLE = new PersonBuilder().withName("Elle Meyer").withPhone("9482224")
-            .withEmail("werner@example.com").withTelegramHandle("@ellemeyer")
+    public static final Person ELLE = new PersonBuilder().withName("Elle Meyer")
+            .withPhone("9482224")
+            .withEmail("werner@example.com")
+            .withTelegramHandle("@ellemeyer")
+            .withModuleName("CS2103T")
             .withContactType("Personal").build();
-    public static final Person FIONA = new PersonBuilder().withName("Fiona Kunz").withPhone("9482427")
-            .withEmail("lydia@example.com").withTelegramHandle("@fionakunz")
+    public static final Person FIONA = new PersonBuilder().withName("Fiona Kunz")
+            .withPhone("9482427")
+            .withEmail("lydia@example.com")
+            .withTelegramHandle("@fionakunz")
+            .withModuleName("CS2101")
             .withContactType("Personal").build();
-    public static final Person GEORGE = new PersonBuilder().withName("George Best").withPhone("9482442")
-            .withEmail("anna@example.com").withTelegramHandle("@georgebest")
+    public static final Person GEORGE = new PersonBuilder().withName("George Best")
+            .withPhone("9482442")
+            .withEmail("anna@example.com")
+            .withTelegramHandle("@georgebest")
+            .withModuleName("CS2109")
             .withContactType("Personal").build();
 
     // Manually added
-    public static final Person HOON = new PersonBuilder().withName("Hoon Meier").withPhone("8482424")
-            .withEmail("stefan@example.com").withTelegramHandle("@hoonmeier")
+    public static final Person HOON = new PersonBuilder().withName("Hoon Meier")
+            .withPhone("8482424")
+            .withEmail("stefan@example.com")
+            .withTelegramHandle("@hoonmeier")
+            .withModuleName("CS2106")
             .withContactType("Work").build();
-    public static final Person IDA = new PersonBuilder().withName("Ida Mueller").withPhone("8482131")
-            .withEmail("hans@example.com").withTelegramHandle("@idamueller")
+    public static final Person IDA = new PersonBuilder()
+            .withName("Ida Mueller")
+            .withPhone("8482131")
+            .withEmail("hans@example.com")
+            .withTelegramHandle("@idamueller")
+            .withModuleName("ST1131")
```

Comment:
Good that you use different ModuleNames for every person instead of the exact same ModuleName for each person to actually test out the functionality of the ModuleName